stable/aws-cloudwatch-metrics/templates/daemonset.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "aws-cloudwatch-metrics.labels" . | nindent 4 }}
 spec:
+  hostNetwork: {{ .Values.hostNetwork | default false}}
```

Comment:
Please add a `hostNetwork` element to the [`values.yaml`](https://github.com/aws/eks-charts/blob/master/stable/aws-cloudwatch-metrics/values.yaml) file itself.

Simply refer to that value in the daemonset.yaml template here, instead of using a `| default false` statement here.

Reply:
done